azure-pipelines.yml

```diff
@@ -14,3 +14,14 @@ steps:
 
 - script: npm run units
   displayName: Run unit tests
+
+# For CI runs on master, automatically publish packages
+- bash: |
+      echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > .npmrc
```

Comment:
Is this going to show in the logs? Seems like we don't want it to be known.

Reply:
No. Its a secret, and we're just writing it to a file here not printing it out anyways